internal/pkg/libexec/pull.go

```diff
@@ -40,7 +40,7 @@ func PullShubImage(filePath, shubRef string, force, noHTTPS bool) {
 
 // PullOciImage pulls an OCI image to a sif
 func PullOciImage(path, uri string, opts types.Options) {
-	b, err := build.NewBuild(uri, path, "sif", "", "", opts)
+	b, err := build.NewBuild(uri, build.Config{Dest: path, Format: "sif", Opts: opts})
```

Comment:
I would update the copyright in this file: `2018-2019`. *I cant comment that line, github wont let me.*